core/protobuf/src/proto/dataserver/protocol.proto

```diff
@@ -3,6 +3,7 @@ syntax = "proto2";
 package alluxio.proto.dataserver;
 
 import "dataserver/status.proto";
+import "shared/acl.proto";
```

Comment:
@aaudiber I refactored proto file in the first commit of this PR. Then I realized it might be be a problem for back compatibility? What is the recommended approach here?

Reply:
The latest version needs to be able to read protocol buffers from previous versions. I suspect moving a protocol buffer definition across files will break this, but you could test it out to be sure. We cannot do this refactor if it will require users to reformat their journal when upgrading versions.

Reply:
@yuzhu Does this work?

Reply:
yes, it works. I am guessing the reason is that I simply moved the file around, but did not change the formatting of the fields. so it was able to read the journal without problem.  I will do some more testing on the journal compatibility today.

Reply:
@aaudiber I tried setfacl and getfacl on directories and files with the before version, generating some journal entries. The after version was able to read it out fine. I also verified that `bin/alluxio readJournal` was able to parse and print the old journal fine. Is there more testing I should do to verify the backward compatibility?   or are you ok with merging this?

Reply:
Awesome, sounds like protos are smart enough to not care about the path. The test you mentioned should be adequate